lib/logstash/outputs/sqs/patch.rb

```diff
@@ -0,0 +1,18 @@
+# This patch was stolen from logstash-plugins/logstash-output-s3#102.
```

Comment:
I'm not sure who maintains the AWS mixin, but it might be a good idea. Will raise it with the powers that be. @ph ?